--- openmp-3.6.0.src/runtime/CMakeLists.txt.orig	2015-03-22 07:29:25.543288244 +0100
+++ openmp-3.6.0.src/runtime/CMakeLists.txt	2015-03-22 07:37:02.083269086 +0100
@@ -51,7 +51,7 @@
 # Build Configuration
 set(os_possible_values          lin mac win mic)
 set(arch_possible_values        32e 32 arm ppc64 aarch64)
-set(build_type_possible_values  release debug relwithdebinfo)
+set(build_type_possible_values  release debug relwithdebinfo pld)
 set(omp_version_possible_values 40 30)
 set(lib_type_possible_values    normal profile stubs)
 set(mic_arch_possible_values    knf knc)
